sgf-viewer/prepare_data/prepare_data.py

```diff
@@ -65,6 +65,22 @@ def run_cmd(cmd, shell=False, dry_run=False):
                         ]
                         run_cmd(shell_cmd, shell=True, dry_run=False)
                         games_count += len(files)
+            for path_with_line in section.get("paths_with_line_num", []):
```

Comment:
the SGF files I'm looking at have several SGFs per file, each on one line, so I'm having the JSON in `content.ts` specify both a filename and a line number